### PR TITLE
Added contents for games page

### PR DESCRIPTION
**Pull request by junxjr**
### Title
Added Contents for Games Page

### Issue Number
Closes Issue #153 

### Description
Added contents for games page. Drop down menu for single player and multi-player games. There are 6 games included for each mode. Games can be selected and confirmed. A pop up window message will be displayed to notify the user that the game is loading and will be opening soon, but it won't actually load since there's no contents linked to the games. Some static images are included to display the games. 

### AI Use
I also used AI for formatting and documentation, and also for setting up the files, but I had to change a lot since it won't work at first.

### Testing
I will be working on the test cases for the components after this PR gets merged. 

### Possible Errors
I don't see any errors right now, but if you run into something, please let me know.

### Image
Here are the screenshots of the games page and the pop up message. 
image
image
image

### CODE REVIEW

File: small-talk/app/components/gamesPage/VisualGameComponent.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+
+// ai-gen start (ChatGPT-3.5, 2)
```
Comment (aiharc1) [inline small-talk/app/components/gamesPage/VisualGameComponent.js:3]: Nice job with documenting GPT usage

**Review by aiharc1 — APPROVED** [review]
Nice work on the games Jun! Looks great

**Review by nguyenj21 — APPROVED** [review]
Nice work on the images and making sure they are not copyrighted. In the future, maybe we can make the images animated when hovering over them if possible? Looks good.

**Review by 39otsu — APPROVED** [review]
Great start on the games. If there's time let's implement an RNG engine!

**Pull request merged**